controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -151,6 +152,20 @@ public ConfigurationFile(final File configurationDir, final String rawName, fina
         this(configurationDir, rawName, name, persistOriginal ? InteractionPolicy.STANDARD : InteractionPolicy.READ_ONLY, false);
     }
 
+    /**
+     * Creates a new ConfigurationFile.
+     *
+     * @param configurationDir  directory in which configuration files are stored. Cannot be {@code null} and must exist
+     *                          and be a directory
+     * @param rawName           default name for configuration files of the type handled by this object.
+     *                          Cannot be {@code null} or an empty string
+     * @param name              user provided name of the configuration file to use
+     * @param interactionPolicy policy governing interaction with the configuration file.
+     */
+    public ConfigurationFile(final File configurationDir, final String rawName, final String name, final InteractionPolicy interactionPolicy, final boolean useGit) {
```

Comment:
Please add javadoc for the useGit param.

Reply:
javadoc added